.travis.yml

```diff
@@ -58,5 +58,14 @@ matrix:
       script: ./script/test manifest
       env: NAME="manifest"
 
+    # python tests
+    - language: python
+      python:
+        - "2.7"
+        - "3.6"
```

Comment:
looking at the CI results, it looks like the tests are only run for python v2.7.  I think that due to the way licensed uses an inclusion matrix, and the solution in this file for testing multiple versions of Go, you need to specify the test suite independently for each python version.

Reply:
👍